lib/src/room.dart

```diff
@@ -291,25 +291,19 @@ class Room {
   /// that you have the room members, call and await `Room.loadHeroUsers()`
```

Comment:
is this comment still valid?

Reply:
Yes it is. The method also makes sure that the directChatMatrixId user is loaded